package.json

```diff
@@ -12,5 +12,33 @@
     "adapt-core"
   ],
   "license": "GPL-3.0",
-  "bugs": "https://github.com/adaptlearning/adapt-contrib-core/issues"
+  "bugs": "https://github.com/adaptlearning/adapt-contrib-core/issues",
+  "private": true,
+  "devDependencies": {
+    "@semantic-release/git": "^10.0.1",
+    "conventional-changelog-eslint": "^3.0.9",
+    "semantic-release": "^19.0.3"
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
+      "@semantic-release/github",
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "eslint"
+        }
+      ],
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
```

Comment:
I'm not sure you need the assets section. That would attach the package.json file to the release as an attachment, i believe?

Reply:
You may not even need the git plugin? https://github.com/semantic-release/git

Reply:
https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository This comment is conflicting

Reply:
Yeah my understanding here was that for commit abilities, the @semantic-release/git package needed to be used, and 'release' files/folders needed specified in the assets array. i.e files that will be updated post-merge, pre-tag